tests/foreman/ui/test_errata.py

```diff
@@ -284,29 +285,6 @@ def test_positive_apply_for_all_hosts(self):
                     self.assertIsNotNone(self.contenthost.package_search(
                         client.hostname, FAKE_2_CUSTOM_PACKAGE))
 
-    @stubbed()
-    @tier2
-    def test_positive_view(self):
```

Comment:
I'd rather cover this test, but modified it a bit - instead of checking each column is present on its own place (which is hard in our case), i'd check all the columns after errata id (brothers of node) contain expected values. This way we'll cover fields are present, values are correct, the only thing that won't be covered - whether fields are on their own place, but some coverage is definitely better than no coverage at all.